src/resources/breaking-changes.md

```diff
@@ -0,0 +1,818 @@
+---
+title: Breaking changes
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists the breaking changes in Dart, organized by release and area.
+Complete release notes are available in the [Dart SDK changelog][changelog].
+
+According to Dart's [breaking change policy][], each listed change contains a link to more
```

Comment:
This sentence reads a little odd to me.  It's not really according the breaking change policy that there is a link.  Perhaps restructure this something like:

The Dart language policy and process around breaking changes and deprecations is described in the [breaking change policy][] document.  In order to help Dart programmers understand and manage the impact of language and library breaking changes and deprecations, all such changes (both upcoming and previously released) are listed on this page.  Each listing contains a link ....